doc/architecture/reference_points_coordinate_systems.adoc

```diff
@@ -0,0 +1,16 @@
+= Reference points and coordinate systems
+
+With Open Simulation Interface, an object's position coordinates refer to the center of the object's bounding box, independent of the coordinate system. This reference point is not identical to the origin of any coordinate systems.
```

Comment:
"...independent of the coordinate system." -> can this be dropped? 

I have a hard time to understand the second sentence. Do you mean: "The reference point does not have to be identical to the origin of sensor coordinates or object coordinates"? 

Would it make sense to shift the two sentences under "World coordinates"?